apps/meeting/src/meetingConfig.ts

```diff
@@ -22,12 +22,9 @@ const postLogConfig = {
   logLevel: SDK_LOG_LEVELS.info,
 };
 
-const enableWebAudio = true;
-
 const config = {
   logLevel,
-  postLogConfig,
-  enableWebAudio,
```

Comment:
I'm not sure, but is this a breaking change?

Reply:
Yes, this is not a breaking change the meetingManagerConfig has this defaulted to `false`.
https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/providers/MeetingProvider/index.tsx#L53